openfisca_dummy_country/parameters/param_root.xml

```diff
@@ -2,14 +2,22 @@
   <NODE code="impot">
     <!-- A basic ongoing parameter -->
     <CODE code="taux" description="Taux d'impôt sur les salaires" format="percent">
-      <VALUE deb="2016-01-01" fuzzy="true" valeur="0.35" />
-      <VALUE deb="2015-01-01" fin="2015-12-31" valeur="0.32" />
-      <VALUE deb="1998-01-01" fin="2014-12-31" valeur="0.3" />
+      <VALUE deb="2016-01-01" valeur="0.35" />
+      <VALUE deb="2015-01-01" valeur="0.32" />
+      <VALUE deb="1998-01-01" valeur="0.3" />
     </CODE>
-    <!-- A basic interrupted parameter -->
+    <!-- A basic parameter which value is expected to change -->
```

Comment:
Erg, this parameter is used by other packages as an example of an interrupted parameter. Why not keep `bouclier` as it is, and make `isf` the parameter with an expected value change ?